keps/sig-storage/2589-csi-migration-portworx/README.md

```diff
@@ -90,3 +136,159 @@ N/A
 ##### e2e tests
 
 - `sig-storage` `Driver: portworx-volume` To ensure the implementation correctness, I/we have manually run the e2e tests, [located in the main k8s repository](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/storage/drivers/in_tree.go). Test results are attached to the pull requests 
+
+
+### Graduation Criteria
+
+* Alpha in 1.23 provided all tests are passing.
+* All functionality is guarded by alpha `CSIMigrationPortworx` feature gate.
+* Beta in 1.31 with design validated by customer deployments
+  (non-production)
+* Manual tests with in-tree Portworx volumes
+* GA in 1.33, with `CSIMigrationPortworx` feature gate graduating to GA.
+
+
+### Upgrade / Downgrade Strategy
+
+N/A
+
+### Version Skew Strategy
+
+N/A
+
+## Production Readiness Review Questionnaire
+
```

Comment:
For PRR, I think we should be consistent and reference the same core CSI Migration KEP as other vendor CSI Migration KEPs:
https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/1885-csi-migration-azurefile#production-readiness-review-questionnaire